### PR TITLE
Vanilla material stat rebalanced

### PR DESCRIPTION
**Pull request by SamaelGray**
## Changes

- Normalized the material mass stat factors for wood and jade.
- Removed the damage multiplier and cooldown time patches from materials, effectively reverting the values to the vanilla ones.
- Normalized the AP factor values.

## Reasoning

- Bioferrite is decided to be kept as is in regards to melee cooldown and damage. There is probably the mass factor that needs to be added to it.
- The vanilla cooldown and damage factors are good enough in vanilla, plasteel gives a better edge resulting in more damage and AP, uranium is slightly slower to swing but good for blunt attacks. Plasteel is slightly better that uranium for blunt weapons in CE which is not good...

## Alternatives

- Add the mass factor patches for stone blocks.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/Core/ThingDefs_Items/Items_Resource_Stuff.xml [hunk 1]
```diff
@@ -231,25 +227,18 @@
...
-		<value>
-			<SharpDamageMultiplier>0.7</SharpDamageMultiplier>
-		</value>
-	</Operation>
-
 	<Operation Class="PatchOperationReplace">
 		<xpath>Defs/ThingDef[defName="WoodLog"]/statBases/BluntDamageMultiplier</xpath>
 		<value>
-			<BluntDamageMultiplier>0.5</BluntDamageMultiplier>
+			<BluntDamageMultiplier>0.8</BluntDamageMultiplier>
```
Comment (N7Huntsman) [inline Patches/Core/ThingDefs_Items/Items_Resource_Stuff.xml:233]: Given that we're making wood sharp damage significantly worse by putting it back to the vanilla sharp damage factor (0.4), we might as well just get rid of this patch and let it go back to the vanilla blunt damage multi of 0.9.

File: Patches/Core/ThingDefs_Items/Items_Resource_Stuff.xml [hunk 2]
```diff
@@ -335,20 +317,6 @@
 		</value>
 	</Operation>
 
-	<Operation Class="PatchOperationReplace">
-		<xpath>Defs/ThingDef[defName="Jade"]/stuffProps/statFactors/MeleeWeapon_CooldownMultiplier</xpath>
-		<value>
-			<MeleeWeapon_CooldownMultiplier>1.15</MeleeWeapon_CooldownMultiplier>
```
Comment (N7Huntsman) [inline Patches/Core/ThingDefs_Items/Items_Resource_Stuff.xml:341]: I'd just leave this--jade is already fairly hopeless as a weapon material (and I don't think you can make much aside from clubs in vanilla), it doesn't need _another_ 15% cooldown.

Moreover, it doesn't make much sense that uranium (almost 6 times denser than jade) has a 110% cooldown, while jade has 130%.

File: Patches/Core/ThingDefs_Items/Items_Resource_Stuff.xml [hunk 3]
```diff
@@ -335,20 +317,6 @@
...
-		</value>
-	</Operation>
-
-	<Operation Class="PatchOperationAdd">
-		<xpath>Defs/ThingDef[defName="Jade"]/stuffProps/statFactors</xpath>
-		<value>
-			<Mass>1.3</Mass>
```
Comment (N7Huntsman) [inline Patches/Core/ThingDefs_Items/Items_Resource_Stuff.xml:348]: By rights, we should actually be setting this to about 0.41 - 0.45.

**Review by N7Huntsman — COMMENTED** [review]
What're the specific concerns regarding the mass factors that you want to "normalize" them? As it stands, the mass factors are set fairly closely to the real-world density of the material (for the real metals, at least).

**Comment by SamaelGray** [comment]
Quoting from the reasoning section: `The mass factors impact things outside combat, CE changing them drastically will bleed the effect into things unrelated to the mod. Also the vanilla game does not have a mass impact by materials. Steel is taken as the baseline and the mass difference of the rest of the materials is halved (except wood which is doubled, it's much lighter irl than ingame) and inserted as the factor.`

**Comment by N7Huntsman** [comment]
I saw the reasoning section, I was just looking for direct examples of our mass factor changes have a meaningful gameplay impact to judge by. Barring something compelling there, I don't see a good reason to move away from using the specific gravity of the material (referencing steel) where it's easy to come by. Jade and Wood sort of get a pass since their sort of weird materials with somewhat limited combat utility in the first place.

Suggested changes to the various damage/cooldown/armor pen factors look fine.

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**